dvc/exceptions.py

```diff
@@ -270,3 +270,18 @@ def __init__(self, path):
         super(OutputFileMissingError, self).__init__(
             "Can't find {} neither locally nor on remote".format(path)
         )
+
+
+class DvcIgnoreInCollectedDirError(DvcException):
+    def __init(self, ignore_dirname):
+        super(DvcIgnoreInCollectedDirError, self).__init__(
+            ".dvcignore file should not be in collected dir path: '{"
+            "}'".format(ignore_dirname)
```

Comment:
This is a strange string break)

Reply:
Sorry, didn't review changes introduced by black